### PR TITLE
Added code samples, coverage [API-665]

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: cached_credentials.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import logging
```
Comment (mdumandag) [inline cached_credentials.py:1]: Why do we have the same file in `examples` folder?

File: examples/keytab_credentials.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    handler.setFormatter(formatter)
+    lg.addHandler(handler)
+
+
+customize_logger("hazelcast")
+customize_logger("hzkerberos")
+
+
+def main():
+    # Create a token provider which uses a principal and its password.
```
Comment (mdumandag) [inline examples/keytab_credentials.py:23]: I think the comments on this and the password credentials sample are mixed. We are mentioning the password in keytab example.

File: keytab_credentials.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import logging
```
Comment (mdumandag) [inline keytab_credentials.py:1]: Why do we have this duplicate file?

File: password_credentials.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import logging
```
Comment (mdumandag) [inline password_credentials.py:1]: Why do we have this duplicate file?

File: cached_credentials.py (hunk 1 above)
Comment (yuce) [reply cached_credentials.py]: Those were added by accident. Removed them.

**Review by mdumandag — APPROVED** [review]

**Pull request merged**